### ISSUE TITLE
Error: estimator is not supported (Keras)

### ISSUE THREAD

**Issue by IITGoaPyVidya** [eli5-org/eli5#37]
Hello community, 

I am using eli5 library for image explanations, I am referring [this](https://eli5.readthedocs.io/en/latest/tutorials/keras-image-classifiers.html) tutorial, but at the end I am facing estimator is not supported, 
Please help me to solve this error, 

any help is highly appreciated,
@lopuhin 

Thanks!

**Comment by lopuhin** [comment]
Hi, this is likely to be due to us only supporting TF 1.x, see #47

**Comment by lopuhin** [comment]
This is solved in #58 and released as 0.16
